mdal/frmts/mdal_ugrid.hpp

```diff
@@ -45,6 +46,12 @@ namespace MDAL
       std::string mMesh2dName;
       std::string mMesh1dName;
       std::string nodeZVariableName() const;
+
+      void writeDimensions( MDAL::Mesh *mesh );
+      void writeVariables( MDAL::Mesh *mesh );
+      void writeGlobals();
+      int faceVerticesMaximumCount() const override
+      { return CFDimensions::Type::MaxVerticesInFace; }
```

Comment:
this is just emum value for CFDimensions::Type::MaxVerticesInFace (so maybe 3 or 4 or whatever is the index. of this enum option in the enum). You want probably mDimenstions.size(CFDimensions::Type::MaxVerticesInFace); nevertheless, why the base function from `MDAL::Mesh` is not good enough?